File: AMS_产品管理组件/code/client/source/YssProductInfo.Support/Plan/Pojo/DvaItem.cs
using Newtonsoft.Json;
using FAST.Common.Service.Pojo.Base;
using FAST.Common.Service.Datastructure;

namespace YssProductInfo.Support.Plan.Pojo
{
    /// <summary>
    /// 核算业务字典pojo
    /// </summary>
    [NodeDesc(ParentNode = "C_DVA_ITEM_CODE_P", TreeNode = "C_DVA_ITEM_CODE")]
    public class DvaItem : BasePojo
    {
        /// <summary>
        /// 核算项目代码
        /// </summary>
        private string c_DVA_ITEM_CODE;

        /// <summary>
        /// 核算项目名称
        /// </summary>
        private string c_DVA_ITEM_NAME;

        /// <summary>
        /// 父级代码
        /// </summary>
        private string c_DVA_ITEM_CODE_P;

        /// <summary>
        /// 序号
        /// </summary>
        private int n_ORDER;

        /// <summary>
        /// 是否明细项 1:明细项 0:非明细项
        /// </summary>
        private int n_DETAIL;

        /// <summary>
        /// 属性: C_DVA_ITEM_CODE 
        /// </summary>
        [JsonProperty(PropertyName = "c_DVA_ITEM_CODE")]
        public string C_DVA_ITEM_CODE
        {
            set { c_DVA_ITEM_CODE = value; }

            get { return c_DVA_ITEM_CODE; }
        }

        /// <summary>
        /// 属性: C_DVA_ITEM_NAME 
        /// </summary>
        [JsonProperty(PropertyName = "c_DVA_ITEM_NAME")]
        public string C_DVA_ITEM_NAME
        {
            set { c_DVA_ITEM_NAME = value; }

            get { return c_DVA_ITEM_NAME; }
        }

        /// <summary>
        /// 属性: C_DVA_ITEM_CODE_P 
        /// </summary>
        [JsonProperty(PropertyName = "c_DVA_ITEM_CODE_P")]
        public string C_DVA_ITEM_CODE_P
        {
            set { c_DVA_ITEM_CODE_P = value; }

            get { return c_DVA_ITEM_CODE_P; }
        }

        /// <summary>
        /// 属性: N_ORDER 
        /// </summary>
        [JsonProperty(PropertyName = "n_ORDER")]
        public int N_ORDER
        {
            set { n_ORDER = value; }

            get { return n_ORDER; }
        }

        /// <summary>
        /// 属性: N_DETAIL 
        /// </summary>
        [JsonProperty(PropertyName = "n_DETAIL")]
        public int N_DETAIL
        {
            set { n_DETAIL = value; }

            get { return n_DETAIL; }
        }

    }
}

File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/sys/dvaitem/pojo/DvaItem.java
package com.yss.ams.base.information.support.sys.dvaitem.pojo;

import com.yss.framework.api.common.co.BasePojo;

public class DvaItem extends BasePojo {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	/**
	 * 核算项目代码
	 */
	private String c_DVA_ITEM_CODE = "";
	
	/**
	 * 核算项目名称
	 */
	private String c_DVA_ITEM_NAME = "";
	
	/**
	 * 父级代码
	 */
	private String c_DVA_ITEM_CODE_P = "";
	
	/**
	 * 序号
	 */
	private int n_ORDER = 0;
	
	/**
	 * 是否明细项 1:明细项 0:非明细项
	 */
	private int n_DETAIL = 0;

	public String getC_DVA_ITEM_CODE() {
		return c_DVA_ITEM_CODE;
	}

	public void setC_DVA_ITEM_CODE(String cDVAITEMCODE) {
		c_DVA_ITEM_CODE = cDVAITEMCODE;
	}

	public String getC_DVA_ITEM_NAME() {
		return c_DVA_ITEM_NAME;
	}

	public void setC_DVA_ITEM_NAME(String cDVAITEMNAME) {
		c_DVA_ITEM_NAME = cDVAITEMNAME;
	}

	public String getC_DVA_ITEM_CODE_P() {
		return c_DVA_ITEM_CODE_P;
	}

	public void setC_DVA_ITEM_CODE_P(String cDVAITEMCODEP) {
		c_DVA_ITEM_CODE_P = cDVAITEMCODEP;
	}

	public int getN_ORDER() {
		return n_ORDER;
	}

	public void setN_ORDER(int nORDER) {
		n_ORDER = nORDER;
	}

	public int getN_DETAIL() {
		return n_DETAIL;
	}

	public void setN_DETAIL(int nDETAIL) {
		n_DETAIL = nDETAIL;
	}
}
